tests/epyccel/test_bitwise.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+import numpy as np
+
+from pyccel.epyccel import epyccel
+from modules import bitwise
+from conftest import *
+
+@pytest.mark.parametrize("a, b",[(True, False),(True, True)])
+def test_right_shift_b_b(language, a, b):
+    f1 = bitwise.right_shift_b_b
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(1, 1),(1, 2)])
+def test_right_shift_i_i(language, a, b):
+    f1 = bitwise.right_shift_i_i
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(True, 2),(True, 1)])
+def test_right_shift_b_i(language, a, b):
+    f1 = bitwise.right_shift_b_i
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(True, 1),(True, 2)])
+def test_left_shift_b_i(language, a, b):
+    f1 = bitwise.right_shift_b_i
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(1, 1),(1, 2)])
+def test_left_shift_i_i(language, a, b):
+    f1 = bitwise.right_shift_i_i
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("a, b",[(True, False),(True, True)])
+def test_left_shift_b_b(language, a, b):
+    f1 = bitwise.right_shift_b_b
+    f2 = epyccel( f1, language = language, verbose = True )
+    assert f1(a, b) == f2(a, b)
+    assert(type(f1(a, b)) == type(f2(a, b))) # pylint: disable=unidiomatic-typecheck
+
+@pytest.mark.parametrize("language",['c',pytest.param('fortran', marks = pytest.mark.xfail(reason="problem in wrapping"))])
```

Comment:
Please pass the `pytest.mark.c` mark for the c case and the `pytest.mark.fortran` mark for the fortran case. 
E.g:
```
@pytest.mark.parametrize( 'lang', (
        pytest.param("fortran", marks = pytest.mark.fortran),
        pytest.param("c", marks = [
            pytest.mark.xfail(reason="zip and functional for not implemented"),
            pytest.mark.c]
        )
    )
)
```

Reply:
I'am not sure why are we marking C test, while it's not an xfail test

Reply:
The marks let us run all the tests which have that mark with one command. E.g to run all c tests in the tests/epyccel folder you need to write `pytest tests/epyccel -m c`, to run all tests you write `pytest tests/epyccel`, to run all fortran tests you write `pytest tests/epyccel -m fortran` (although this will only catch the tests that can also run in c as we did not add this mark systematically from the beginning). The marker is added to all c tests in tests/epyccel/conftest.py
If you don't mark the test then it will not be grouped with the other c tests on travis. When we run the tests we run
```
  - ${PYTHON} -m pytest -rx -m "not parallel and c" --ignore=ast --ignore=printers --ignore=symbolic
  - ${PYTHON} -m pytest -rx -m "not parallel and not c" --ignore=ast --ignore=printers --ignore=symbolic
```
If it is not marked then it will be in the group "not c"

Reply:
See https://docs.pytest.org/en/stable/mark.html

Reply:
I didn't see the fixture that marks all c tests and fortran tests, so i assumed we are marking only xfail tests. So basically only when we override the language argument we should also add the marker.

Reply:
Yes exactly